io/include/pcl/io/file_io.h

```diff
@@ -298,7 +280,8 @@ namespace pcl
   {
     Type value;
     memcpy (&value, &cloud.data[point_index * point_size + cloud.fields[field_idx].offset + fields_count * sizeof (Type)], sizeof (Type));
-    if (!std::isfinite (value))
+    //MSVC is missing bool std::isfinite(IntegralType arg); variant, so we need to cast ourself to double
```

Comment:
Can we create two versions just like with `copyValueString`? The integral version won't even need to `memcpy`, just return `true`.